identity-iota/src/did/doc/iota_document.rs

```diff
@@ -173,6 +218,16 @@ impl IotaDocument {
     unsafe { IotaDID::new_unchecked_ref(self.document.id()) }
   }
 
+  /// Returns a reference to the `IotaDocument` controller.
+  pub fn controller(&self) -> Option<&IotaDID> {
+    unsafe { self.document.controller().map(|d| IotaDID::new_unchecked_ref(d)) }
```

Comment:
This should have a `SAFETY` comment to explain how this is considered safe. doesn't really need to be as verbose as the others and could just reference `check_authentication`

*edit:* noticed there were quite a few so definitely feel free to keep it short 😅

Reply:
Done.